src/getting-started/installation.md

```diff
@@ -1,45 +1,80 @@
-
-## Optional Features
 
-As ratatui grows and evolves, this list may change, so make sure to check the [main repo](./https://github.com/ratatui-org/ratatui) if you are unsure.
+And you can start programming TUIs.
 
-### Feature: Backend Selection
+`ratatui` is a library (i.e. a Rust crate).
+Running the above command in your console will add the latest version of `ratatui` to your project.
 
-See [Choosing a Backend](./ch01-02-choosing-a-backend.md) for more information. However, for most cases, the default `crossterm` backend is enough.
+If you are interested in adding a specific version, you can run the following:
 
 ```console
-# Defaults to crossterm
-cargo add ratatui
-# Unset the default crossterm feature and select one of the other backends
-cargo add ratatui --no-default-features --features=terminon 
-cargo add ratatui --no-default-features --features=termwiz
+cargo add ratatui --version 0.19.0
 ```
 
-### Feature: All-Widgets
+## Quickstart
+
+To start with a new project, you can run the following:
 
-This feature enables some extra widgets that are not in `default` to save on compile time. As of v0.21, the only widget in this feature group is the `calendar` widget, which can be enabled with the `widget-calendar` feature.
-```console
-cargo add ratatui --features all-widgets
+```
+mkdir project
+cd project
+cargo init
+cargo add ratatui crossterm
 ```
 
-### Feature: Widget-Calendar
-This feature enables the calendar widget, which requires the `time` crate. 
+```admonish note
+`ratatui` has to be combined with a terminal backend.
+You can learn more about the different terminal backends in [section on choosing a backend](./choosing-a-backend.md).
+For the examples in this book, we are going to use `crossterm`.
+```
 
-```console
-cargo add ratatui --features widget-calendar
+Modify `src/main.rs` to the following:
+
+```rust,no_run,noplayground
+use ratatui::{
+  prelude::{CrosstermBackend, Terminal},
+  widgets::Paragraph,
+};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+  crossterm::terminal::enable_raw_mode()?;
+  crossterm::execute!(std::io::stderr(), crossterm::terminal::EnterAlternateScreen)?;
+
+  let mut terminal = Terminal::new(CrosstermBackend::new(std::io::stderr()))?;
+
+  loop {
+    terminal.draw(|f| {
+      f.render_widget(Paragraph::new("Hello World! (press 'q' to quit)"), f.size());
+    })?;
+
+    if let crossterm::event::Event::Key(key) = crossterm::event::read()? {
+      if key.code == crossterm::event::KeyCode::Char('q') {
+        break;
+      }
+    }
+  }
+
+  crossterm::execute!(std::io::stderr(), crossterm::terminal::LeaveAlternateScreen)?;
+  crossterm::terminal::disable_raw_mode()?;
+
+  Ok(())
+}
 ```
 
-### Feature: Serde
+Then you can run it with:
 
-```console
-cargo add ratatui --features serde
 ```
+cargo run
+```
+
+Press `q` to exit.
+
+![](https://user-images.githubusercontent.com/1813121/262239086-9c483c12-75ba-409a-b97b-8faed4e9e468.gif)
```

Comment:
The source listing goes by a bit fast to actually read it, and that listing is available above. Perhaps just show the cargo run step here and link to the vhs tape and instructions to update.

In general, I've been trying to reduce the frantic-ness of the examples as fast text flashing / changing can be jarring - it's a balancing act though.

Reply:
Resolved in https://github.com/ratatui-org/ratatui-book/pull/15/commits/fd62fbe78ff9899f04bc12c8cf85fc15259d7ab9.